test_ir.py

```diff
@@ -37,5 +37,68 @@ class DisplaySystem(jkt.KernelTests):
     {'code': '1:3', 'mime': {'text/plain':'[1] 1 2 3'}},
     ]
 
+class AbstractIRKernel(jkt.KernelTests):
```

Comment:
On 23 May 2016 at 16:43, Jan Schulz <notifications@github.com> wrote:

> My intention was that the kernel was restarted for each test... as I play
> with options it seems to do that.
>
OK. By default JKT reuses a kernel between tests, because starting it up
and spinning it down can be slow, and the tests are written not too depend
on order. If you want to create a new kernel for each test, I'd recommend
using setUp and tearDown methods - essentially equivalent to the setUpClass
and tearDownClass methods that are already there:
https://github.com/jupyter/jupyter_kernel_test/blob/6c392ddf54a24c464fa5e07ad7fb001023f04698/jupyter_kernel_test/__init__.py#L21